appveyor.yml

```diff
@@ -15,15 +25,43 @@ before_test:
     python -m pip install -r requirements.txt
     python -m pip install -r requirements-devel.txt
     python -m pip install --prefix %VIRTUAL_ENV% -e .
+
 test_script:
 - cmd: |
     venv\\Scripts\\activate.bat
     ECHO %VIRTUAL_ENV%
-    python -m unittest run tests.integration.store.test_store_login_logout -v
+
 after_test:
 - cmd: |
     venv\\Scripts\\activate.bat
     ECHO %VIRTUAL_ENV%
-    python setup.py sdist
+
+    ECHO "Building snapcraft.exe..."
+    pyinstaller.exe --onefile snapcraft.spec
+
+    ECHO "Test signing snapcraft.exe..."
+    powershell.exe windows\\generate-self-signed-cert.ps1
+    "%SIGNTOOL%" sign /fd SHA256 /T "http://timestamp.verisign.com/scripts/timstamp.dll" /F test-signing.pfx /P Password1234 dist\\snapcraft.exe
+
+    ECHO "Downloading multipass installer..."
```

Comment:
do we need this for testing?
Actually, this could probably be faked for the tests :-)

Reply:
The installer logic does require the installer to be bundled, but yes, it could be faked with some echo exe or something.

I there is value to having multipass included if someone wants to download and install/test the artifact itself?

Reply:
I removed multipass bundling altogether.